cloudevents/extensions/loglevel.md

```diff
@@ -0,0 +1,153 @@
+# Log Level Extension
+
+
+## Attributes
+
+### loglevelname 
+
+- Type: `String`
+- Description: Human readable representation of the log level
+- Constraints
+  - OPTIONAL
+  - if present, MUST be a non-empty string
+  - SHOULD be lowercase
+  - RECOMMENDED values include   `emergency`, `alert` , `critical` ,`error`, `warning`, `notice`, `info`, `debug`, and
```

Comment:
Minor formatting things...
- look for odd spacing, like extra before `emergency`, before the `,` on the next line, at the start of the next line (one), one missing from the start of line 17....
- can you wrap lines at 80 chars to match the other docs?